math.js

```diff
@@ -96,6 +96,13 @@ function profilePage(img){
     return "image"
 }
 
+
+/**
+ * settings page function documentation.
+ * @param {number} a 
```

Comment:
can you please correct your commit meassage

Reply:
sorry, added the correct commit message.